### PR TITLE
add images

### PR DESCRIPTION
**Pull request by genti94**

### CODE REVIEW

File: docs/tutorial/basic-content-editing.rst [hunk 1]
```diff
@@ -22,16 +22,22 @@ If not, add ``/?edit`` to the end of the URL to trigger the login prompt.
 Using Draft mode
 ================
 
-You're in *Live mode*. Switch to *Draft mode*, using the button in the Toolbar.
+You're in *Live mode*. Switch to *Draft mode*, using the button in the Toolbar.
```
Comment (evildmp) [inline docs/tutorial/basic-content-editing.rst]: It looks like your editor is saving white space at the end of lines (it should not).

Can you adjust the option in the editor that controls this, save the affected files, and push again? Thanks.

**Comment by FinalAngel** [comment]
@genti94 please resolve ;)

File: docs/tutorial/create-new-project.rst [hunk 2]
```diff
@@ -73,4 +76,5 @@ will be unaffected. Repeat the deployment. If it fails again, contact the Aldryn
 either `by email <mailto:support@aldryn.com>`_ or by using the *Intercom tool* on any page on the
 Aldryn website.
 
-.. todo:: an image for the Intercom widget
+.. image:: images/intercom.png
```
Comment (evildmp) [inline docs/tutorial/create-new-project.rst]: Can we have an image that shows the location of the tool in a page please? So show most of the bottom of a page with the tool highlighted, or whatever looks best.

File: docs/tutorial/basic-content-editing.rst [hunk 3]
```diff
@@ -24,14 +24,20 @@ Using Draft mode
 
 You're in *Live mode*. Switch to *Draft mode*, using the button in the Toolbar.
 
-.. todo:: an image for the toolbar with the live/draft mode switch highlighted
+.. image:: images/live.png
+   :alt: live
 
 In *Draft mode*, you're able to change much of the site's content simply by double-clicking on it.
 
+.. image:: images/draft.png
```
Comment (evildmp) [inline docs/tutorial/basic-content-editing.rst]: The draft button image seems to be horizontally distorted.

File: docs/tutorial/collaborating.rst [hunk 4]
```diff
@@ -43,7 +45,8 @@ update content on a day-to-day basis.
 
 To invite *Clients (CMS-only users)*, provide their email addresses, separated by spaces.
 
-.. todo:: an image of the Invite CMS-only users field with fake @example.com email addresses in it
+.. image:: images/cms_user_field.png
```
Comment (evildmp) [inline docs/tutorial/collaborating.rst]: Whenever you need to give a fake example URL, you should use ``example.com`` as the domain, which exists for that purpose. Please amend.

**Comment by evildmp** [comment]
Really great work @genti94, thanks very much.

I'm merging this into master, with some changes.

For future reference, please note:

* when images are small enough, provide a ``width`` - e.g. ``:width: 50%`` or ``:width: 25%`` and make them ``:align: right``. Then place them before the block of text that you want them to be floated next to
* always make sure you have provided good ``alt`` text where appropriate
* make sure that you use image widths consistently, so for example the *Test it* and *Go live* buttons should be of a similar size
* sometimes it makes sense to put images inline - see the draft/live buttons in ``basic-content-editing.rst``, or the search/upload icons in ``files-and-images.rst``.

Things to do:

* [ ] We also need some slightly different images, but I have commented on these specifically in the file list.
* [ ] Please also provide a new ``icon_tour.png`` image, that shows the location of the icon in the context of the page (i.e.. show enough of the page in the image to show where the icon is).

**Pull request merged**